src/resources/parser/glb-container-resource.js

```diff
@@ -0,0 +1,313 @@
+import { Asset } from '../../asset/asset.js';
+import { Entity } from '../../framework/entity.js';
+import { MeshInstance } from '../../scene/mesh-instance.js';
+import { MorphInstance } from '../../scene/morph-instance.js';
+import { SkinInstance } from '../../scene/skin-instance.js';
+import { SkinInstanceCache } from '../../scene/skin-instance-cache.js';
+import { Model } from '../../scene/model.js';
+
+// Container resource returned by the GlbParser. Implements the ContainerResource interface.
+class GlbContainerResource {
+    constructor(data, asset, assets, defaultMaterial) {
+        const createAsset = function (type, resource, index) {
+            const subAsset = GlbContainerResource.createAsset(asset.name, type, resource, index);
+            assets.add(subAsset);
+            return subAsset;
+        };
+
+        let i;
```

Comment:
Declare in each `for` loop. That's generally how the rest of the engine codebase declares `for` loops now.

Reply:
done